library/src/main/java/fr/tvbarthel/intentshare/IntentShare.java

```diff
@@ -94,6 +99,7 @@ private IntentShare(Context context) {
         packageWithExtraProvider = new ArrayList<>();
         this.listener = null;
         this.iconLoader = new AsyncIconLoader();
+        this.chooserTitle = context.getString(R.string.default_sharing_label);
```

Comment:
going to use resourcePrefix

https://code.google.com/p/android/issues/detail?id=69006